apps/editing-toolkit/editing-toolkit-plugin/jetpack-timeline/blocks/src/timeline-item.js

```diff
@@ -88,6 +109,7 @@ export function registerTimelineItemBlock() {
 										onChange: ( background ) =>
 											setAttributes( { background: background || DEFAULT_BACKGROUND } ),
 										label: __( 'Background Color', 'full-site-editing' ),
+										colors: Colors(),
```

Comment:
The way `Colors` is being used means that it should be a "hook" instead of a component. [The capitalised function name means it's a component](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized), and so you'd expect it to be used like `<Colors />`. But it really just wants to wrap to the `useSetting` hook with its own logic, and so it should be [a custom hook](https://reactjs.org/docs/hooks-custom.html).

So maybe rename `Colors` to something like `useBackgroundColors` or something like that?

To satisfy the linter you'll also need to convince it that the `edit` attribute on line 55 represents a React component. Which you can do with something like:
```javascript
edit: function Edit( { attributes, clientId, setAttributes } ) {
```

Reply:
> The way `Colors` is being used means that it should be a "hook" instead of a component. [The capitalised function name means it's a component](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized), and so you'd expect it to be used like `<Colors />`. But it really just wants to wrap to the `useSetting` hook with its own logic, and so it should be [a custom hook](https://reactjs.org/docs/hooks-custom.html).

@p-jackson I learn something new every day!

> So maybe rename `Colors` to something like `useBackgroundColors` or something like that?
> 
> To satisfy the linter you'll also need to convince it that the `edit` attribute on line 55 represents a React component. Which you can do with something like:

Made this change with 9ad6752f3c